metadata/containers.go

```diff
@@ -236,7 +236,7 @@ func (s *containerStore) Delete(ctx context.Context, id string) error {
 
 func validateContainer(container *containers.Container) error {
 	if err := identifiers.Validate(container.ID); err != nil {
-		return errors.Wrapf(err, "container.ID validation error")
+		return errors.Wrap(err, "container.ID")
```

Comment:
This change seems unneeded

Reply:
within all the validation packages there is an `errdefs.ErrInvalidArgument` and whenever `validateContainer` is called the err message includes `... failed validation`. thought this would help avoid redundancy.